geonode/base/api/views.py

```diff
@@ -428,6 +430,10 @@ def set_perms(self, request, pk=None):
     def set_thumbnail_from_bbox(self, request, resource_id):
         try:
             resource = ResourceBase.objects.get(id=ast.literal_eval(resource_id))
+
+            if isinstance(resource.get_real_instance(), (Document, GeoApp)):
```

Comment:
@mattiagiupponi sorry for being picky today but wouldn't it be better to do 

`if not isinstance(resource.get_real_instance(), (Document, Map)):`

? It's better to say what we support rather then list what is not supported :)